api/jobs.go

```diff
@@ -224,9 +224,14 @@ func (j *Jobs) Deregister(jobID string, purge bool, q *WriteOptions) (string, *W
 }
 
 // ForceEvaluate is used to force-evaluate an existing job.
-func (j *Jobs) ForceEvaluate(jobID string, q *WriteOptions) (string, *WriteMeta, error) {
+func (j *Jobs) ForceEvaluate(jobID string, opts EvalOptions, q *WriteOptions) (string, *WriteMeta, error) {
```

Comment:
Worth renaming to EvaluateWithOpts?

Reply:
I modeled this after `JobDeregisterOptions`.

Reply:
I meant the name of the function. Since it can be more than just force given it takes a generic options object